plugins/source/azure/resources/services/cognitiveservices/account_skus.go

```diff
@@ -0,0 +1,49 @@
+package cognitiveservices
+
+import (
+	"context"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/cognitiveservices/armcognitiveservices"
+	"github.com/apache/arrow/go/v13/arrow"
+	"github.com/cloudquery/cloudquery/plugins/source/azure/client"
+	"github.com/cloudquery/plugin-sdk/v3/schema"
+	"github.com/cloudquery/plugin-sdk/v3/transformers"
+)
+
+func accountSKUs() *schema.Table {
+	return &schema.Table{
+		Name:                 "azure_cognitiveservices_account_skus",
+		Resolver:             fetchAccountSKUs,
+		PostResourceResolver: client.LowercaseIDResolver,
+		Description:          "https://learn.microsoft.com/en-us/rest/api/cognitiveservices/accountmanagement/accounts/list-skus?tabs=HTTP#accountsku",
+		Multiplex:            client.SubscriptionMultiplexRegisteredNamespace("azure_cognitiveservices_account_skus", client.Namespacemicrosoft_cognitiveservices),
+		Transform:            transformers.TransformWithStruct(&armcognitiveservices.AccountSKU{}),
```

Comment:
These don't have an ID. In `azure_compute_skus` we calculate a hash. We could do this here too